hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -1447,6 +1443,12 @@ protected void onClear() {
     }
 
     @Override
+    public void close() {
```

Comment:
I don't know :) Closing cache means, you don't want to access cache anymore without destroying the cache. I mean clearing in-memory is fine but main purpose is not to remove disk data.

So, still do you think we need to invalidate near-cache? If yes, I'll update as you suggested...